app/lib/features/quiz_registration/presentation/pages/quiz_registration_page.dart

```diff
@@ -233,9 +260,14 @@ class _QuizRegistrationPageState extends State<QuizRegistrationPage> {
                       padding: const EdgeInsets.symmetric(horizontal: 40),
                       width: double.infinity,
                       child: Button(
-                        onTap: () =>
-                            setState(() => selectedWeek = 'next_weekly_quiz'),
-                        customButtonColor: Colors.green.shade400,
+                        onTap: () {
+                          if (!isNextWeekSelected) {
+                            setState(() => selectedWeek = 'next_weekly_quiz');
+                          }
+                        },
+                        customButtonColor: isNextWeekSelected
+                            ? Colors.grey
+                            : Colors.green.shade400,
```

Comment:
instead of ganti warna gimana kalau mainan di parameter `isDisabled` mas?
nanti bisa langsung
```
isDisabled: isNextWeekSelected
```
contoh: https://github.com/ia-toki/bebras-pandai/pull/35